packages/fonts/package.json

```diff
@@ -34,6 +34,7 @@
   ],
   "type": "module",
   "exports": {
+    ".": { "style": "./dist/fonts.css" },
     "./fonts.css": "./dist/fonts.css"
```

Comment:
Have you tested on shire directly? I could only make it work with 
```
"exports": {
    ".": {
      "require": "./dist/fonts.css",
      "import": "./dist/fonts.css",
      "default": "./dist/fonts.css"
    },
    "./fonts.css": {
      "require": "./dist/fonts.css",
      "import": "./dist/fonts.css",
      "default": "./dist/fonts.css"
    }
  },
```

Reply:
yes you are right, I will push a new fix soon 😄